client/modules/accounts/containers/auth/loginContainer.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component, PropTypes } from "react";
+import { composeWithTracker } from "/lib/api/compose";
+import AuthContainer from "./authContainer";
+import { ForgotContainer } from "../passwordReset";
+
+class LoginContainer extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentView: props.loginFormCurrentView
+    };
+
+    this.showForgotPasswordView = this.showForgotPasswordView.bind(this);
+    this.showSignUpView = this.showSignUpView.bind(this);
+    this.showSignInView = this.showSignInView.bind(this);
+  }
+
+  showForgotPasswordView(e) {
+    e.preventDefault();
+
+    this.setState({
+      currentView: "loginFormResetPasswordView"
+    });
+  }
+
+  showSignUpView(e) {
+    e.preventDefault();
+
+    this.setState({
+      currentView: "loginFormSignUpView"
+    });
+  }
+
+  showSignInView(e) {
```

Comment:
I think we want to stay standard and just use `event` everywhere. I understand why you are using it here but if the logic gets any more complex it could lead to confusion as e could `event` or it could be `errors`